cluster/common.sh

```diff
@@ -289,11 +289,7 @@ function load-or-gen-kube-basicauth() {
 function set_binary_version() {
   if [[ "${1}" =~ "/" ]]; then
     IFS='/' read -r -a path <<< "${1}"
-    if [[ "${path[0]}" == "release" ]]; then
-      KUBE_VERSION=$(gsutil cat "gs://kubernetes-release/${1}.txt")
-    else
-      KUBE_VERSION=$(gsutil cat "gs://k8s-release-dev/${1}.txt")
```

Comment:
k8s-release-dev is a CI bucket, we still need the two codepaths

Reply:
ok, I can revert. I made this update because looking at accessing the objects from `dl.k8s.io`, it seems like the same object that k8s-release-dev/ci is, given `/ci/latest-1.txt` for example:

```bash
> gsutil ls -L -b gs://k8s-release-dev/ci/latest-1.txt | grep "Hash" 
    Hash (crc32c):          LSpVig==
    Hash (md5):             Vt8SKKffSzxBtV5du7cHbw==

> curl -vL https://dl.k8s.io/ci/latest-1.txt 2>&1 | grep -i 'x-goog-hash:'
  < x-goog-hash: crc32c=LSpVig==
  < x-goog-hash: md5=Vt8SKKffSzxBtV5du7cHbw==

```

Reply:
TIL!

https://github.com/kubernetes/k8s.io/blob/3cd6e23185c896e11b40a484d71280ac54b96d9f/apps/k8s-io/configmap-nginx.yaml#L183

@ameukam something to be aware of for the fastly migration, though I assume we can keep the nginx and just change what it redirects to.

Reply:
Yes. we only gonna change https://github.com/kubernetes/k8s.io/blob/3cd6e23185c896e11b40a484d71280ac54b96d9f/apps/k8s-io/configmap-nginx.yaml#L185 to use the subdomain delegated to Fastly.